Cellpose_Plugin_CellProfiler_Instructions/README.md

```diff
@@ -0,0 +1,97 @@
+# Cellpose Plugin for CellProfiler installation
+
+## For Linux:
+
+### Step 1 
```

Comment:
Consider bumping down the header for consistency with other step headers here.

```suggestion
## Step 1 
```

Reply:
Fixed!